test/end-to-end/verificationLinks/verificationLinks.spec.js

```diff
@@ -0,0 +1,62 @@
+/* global browser, element, by */
+const chai = require('chai');
+const helpers = require('../shared/helpers');
+const FU = require('../shared/FormUtils');
+const GU = require('../shared/GridUtils');
+const components = require('../shared/components');
+const Filters = require('../shared/components/bhFilters');
+const InvoiceRegistryPage = require('../patient/invoice/registry.page.js');
+const expect = chai.expect;
+
+helpers.configure(chai);
+
+describe('Check links', () => {
+  const path = '#!/';
+  const filters = new Filters();
+  const page = new InvoiceRegistryPage();
+
+  before(() => helpers.navigate(path));
+
+
+  function expectNumberOfGridRows(number) {
+    expect(rows.count(),
+      `Expected Patient Registry ui-grid's row count to be ${number}.`
+    ).to.eventually.equal(number);
+  }
+
+  const grid = element(by.id('patient-registry'));
+  const rows = grid.element(by.css('.ui-grid-render-container-body'))
+    .all(by.repeater('(rowRenderIndex, row) in rowContainer.renderedRows track by $index'));  
+
+  it('Check link betwen Patient Groups -> Patient Registry', () => {
+    browser.get('#!/patients/groups');
```

Comment:
I think it would be better to use `helpers.navigate()` since it using hash-based routing.  `browser.get()` may slow down loading the whole page by reloading angular.

The [documenation](http://www.protractortest.org/#/api?view=ProtractorBrowser.prototype.get) is not clear on this though.